internal/extender/overhead.go

```diff
@@ -146,10 +147,21 @@ func (o *OverheadComputer) compute(ctx context.Context) {
 	o.overheadLock.Unlock()
 }
 
-func podToResources(pod *v1.Pod) *resources.Resources {
+func podToResources(ctx context.Context, pod *v1.Pod) *resources.Resources {
+	oneCPU := resource.NewMilliQuantity(1000, resource.DecimalSI)
+	oneGiB := resource.NewQuantity(1*1024*1024*1024, resource.BinarySI)
```

Comment:
can these be constants?

Reply:
They can only be vars, not sure if that's better?

Reply:
moved them outside